src/lib.rs

```diff
@@ -9,13 +9,13 @@
 // TODO (v0.6): Decorate functions with #[must_use]
 //#![warn(clippy::must_use_candidate)]
 #![cfg_attr(not(test), warn(unsafe_code))]
-#![warn(clippy::all)]
-#![warn(clippy::cast_lossless)]
-#![warn(clippy::explicit_deref_methods)]
-#![warn(clippy::explicit_into_iter_loop)]
-#![warn(clippy::explicit_iter_loop)]
-// TODO: Decorate functions with #[must_use]
-//#![warn(clippy::must_use_candidate)]
+#![warn(clippy::pedantic)]
+#![warn(clippy::clone_on_ref_ptr)]
+#![allow(clippy::wildcard_imports)]
```

Comment:
My personal preference is not to use wildcard imports but if everyone else like it, I'm totally fine :wink:

Reply:
:+1: Then we should append `// TODO` as for other temporarily allowed attributes.